### PR TITLE
worldmap: add hunter area tooltips

### LINKED ISSUE #6848: Add hunter area tooltips to world map plugin
Hey,

My friend came up with the idea of showing which hunter level you need for each training area, much like the teleports do.
An extra idea would be showing what can be hunted at the location.

Are these good ideas? Let me know what you think, thanks for your time.

### PR DESCRIPTION
**Pull request by melkypie**
closes #6848

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+		HunterCreature.SAPPHIRE_GLACIALIS, HunterCreature.SNOWY_KNIGHT,
+		HunterCreature.SABRE_TOOTHED_KEBBIT, HunterCreature.SABRE_TOOTHED_KYATT),
+	SLEPE_NORTH(new WorldPoint(3677, 3405, 0), HunterCreature.SWAMP_LIZARD),
+	UZER_HUNTER_AREA(new WorldPoint(3401, 3104, 0), HunterCreature.GOLDEN_WARBLER, HunterCreature.DESERT_DEVIL,
+		HunterCreature.ORANGE_SALAMANDER),
+	WILDERNESS(new WorldPoint(3142, 3771, 0), HunterCreature.BLACK_CHINCHOMPA),
+	;
+
+	private final WorldPoint location;
+	private final HunterCreature[] creatures;
```
Comment (dekvall) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java:73]: This looks unused?

Comment (melkypie) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java:73]: true

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	KARAMJA_HUNTER_AREA(new WorldPoint(2786, 3001, 0), HunterCreature.HORNED_GRAAHK),
+	KEBOS_SWAMP(new WorldPoint(1184, 3595, 0), HunterCreature.CRIMSON_SWIFT),
+	KOUREND_WOODLAND_CENTER(new WorldPoint(1512, 3478, 0), HunterCreature.RUBY_HARVEST),
+	KOUREND_WOODLAND_NORTH_WEST(new WorldPoint(1481, 3504, 0), HunterCreature.CHINCHOMPA),
+	KOUREND_WOODLAND_SOUTH(new WorldPoint(1556, 3436, 0), HunterCreature.COPPER_LONGTAIL),
+	LAKE_MOLCH(new WorldPoint(1363, 3632, 0), HunterCreature.BLUEGILL, HunterCreature.COMMON_TENCH,
+		HunterCreature.MOTTLED_EEL, HunterCreature.GREATER_SIREN),
+	OURANIA_HUNTER_AREA_EAST(new WorldPoint(2447, 3219, 0), HunterCreature.RED_SALAMANDER),
+	OURANIA_HUNTER_AREA_SOUTH(new WorldPoint(2475, 3240, 0), HunterCreature.RED_SALAMANDER),
+	PISCATORIS_HUNTER_aREA(new WorldPoint(2335, 3584, 0), HunterCreature.COMMON_KEBBIT, HunterCreature.COPPER_LONGTAIL,
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java:60]:
```suggestion
	PISCATORIS_HUNTER_AREA(new WorldPoint(2335, 3584, 0), HunterCreature.COMMON_KEBBIT, HunterCreature.COPPER_LONGTAIL,
```

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	FELDIP_HUNTER_AREA(new WorldPoint(2557, 2912, 0), HunterCreature.CRIMSON_SWIFT, HunterCreature.FELDIP_WEASEL,
+		HunterCreature.TROPICAL_WAGTAIL, HunterCreature.SPINED_LARUPIA, HunterCreature.BARB_TAILED_KEBBIT,
+		HunterCreature.BLACK_WARLOCK, HunterCreature.CARNIVOROUS_CHINCHOMPA),
+	FOSSIL_ISLAND1(new WorldPoint(3693, 3800, 0), HunterCreature.HERBIBOAR),
+	FOSSIL_ISLAND2(new WorldPoint(3701, 3809, 0), HunterCreature.HERBIBOAR),
+	FOSSIL_ISLAND3(new WorldPoint(3703, 3829, 0), HunterCreature.HERBIBOAR),
+	FOSSIL_ISLAND4(new WorldPoint(3749, 3850, 0), HunterCreature.HERBIBOAR),
+	FOSSIL_ISLAND5(new WorldPoint(3684, 3870, 0), HunterCreature.HERBIBOAR),
+	FOSSIL_ISLAND_UNDERWATER(new WorldPoint(3743, 10295, 0), HunterCreature.FISH_SHOAL),
+	GWENITH_HUNTER_AREA(new WorldPoint(2269, 3408, 0), HunterCreature.CARNIVOROUS_CHINCHOMPA),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/HunterAreaLocation.java:50]: This is missing the identical spot on the Prifddinas world map.

**Comment by melkypie** [comment]
Fixed both

**Pull request merged**